src/frontend/admin/src/components/templates/organizations/form/sections/OrganizationAddressForm.tsx

```diff
@@ -0,0 +1,147 @@
+import * as React from "react";
+import * as Yup from "yup";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import Box from "@mui/material/Box";
+import Grid from "@mui/material/Unstable_Grid2";
+import { FormattedMessage, useIntl } from "react-intl";
+import Typography from "@mui/material/Typography";
+import {
+  DTOOrganizationAddress,
+  Organization,
+} from "@/services/api/models/Organization";
+import { SimpleCard } from "@/components/presentational/card/SimpleCard";
+import { RHFProvider } from "@/components/presentational/hook-form/RHFProvider";
+import { RHFTextField } from "@/components/presentational/hook-form/RHFTextField";
+import { useOrganizations } from "@/hooks/useOrganizations/useOrganizations";
+import { organizationAddressFormMessages } from "@/components/templates/organizations/form/translations";
+import { RHFSelect } from "@/components/presentational/hook-form/RHFSelect";
+
+type Props = {
+  organization: Organization;
+};
+export function OrganizationAddressForm({ organization }: Props) {
+  const intl = useIntl();
+  const organizationQuery = useOrganizations({}, { enabled: false });
+  const organizationAddress = organization?.addresses?.[0];
+
+  const getDefaultValues = () => {
+    return {
+      title: organizationAddress?.title ?? "",
+      country: organizationAddress?.country ?? "FR",
+      address: organizationAddress?.address ?? "",
+      postcode: organizationAddress?.postcode ?? "",
+      city: organizationAddress?.city ?? "",
+      first_name: organizationAddress?.first_name ?? "",
+      last_name: organizationAddress?.last_name ?? "",
+    };
+  };
+
+  const RegisterSchema = Yup.object().shape({
```

Comment:
Field validation naming issue:

<img width="272" alt="Capture d’écran 2024-02-29 à 11 16 55" src="https://github.com/openfun/joanie/assets/9628870/c2316869-88ce-4906-acb9-c77cae92934f">

Reply:
Yes it is planned in a slightly more global way in another PR